ha/core/config/config_manager.py

```diff
@@ -41,26 +41,33 @@ class ConfigManager:
 
     # TODO: create separate function for log and conf file
     @staticmethod
-    def init(log_name):
+    def init(log_name, log_path=None, level="INFO",
```

Comment:
is log_path variable needed anymore ? since log path is going to come from ha.conf ( and in there its going to be populated from cluster.conf ?)

Reply:
it is good to have as test can provide different path if needed